src/CredentialsCheckFlood.php

```diff
@@ -123,7 +123,7 @@ protected function getAccountIdentifier($ip, $name) {
       $account_by_name = $storage->loadByProperties(['name' => $name]);
       $this->accounts[$name] = reset($account_by_name);
     }
-    /** @var \Drupal\Core\Session\AccountInterface $account **/
+    // @var \Drupal\Core\Session\AccountInterface $account
```

Comment:
@haringsrob I agree this looks a little strange, but the previous was failed by the validator.